src/applications/hca/config/chapters/insuranceInformation/general.js

```diff
@@ -20,8 +20,7 @@ import {
 } from '../../../utils/helpers';
 import { emptyObjectSchema } from '../../../definitions';
 
-const { provider } = fullSchemaHca.definitions;
-const { isCoveredByHealthInsurance } = fullSchemaHca.properties;
+const { provider, isCoveredByHealthInsurance } = fullSchemaHca.properties;
```

Comment:
`provider` is not a defined property anymore. You'll need to rework the declaration on this page to get the required fields array merged in to the `providers` property